src/DropdownMenu.js

```diff
@@ -18,13 +18,15 @@ const propTypes = {
   persist: PropTypes.bool,
   strategy: PropTypes.string,
   container: targetPropType,
+  updateOnSelect: PropTypes.bool,
   right: deprecated(PropTypes.bool, 'Please use "end" instead.'),
 };
 
 const defaultProps = {
   tag: 'div',
   flip: true,
   modifiers: [],
+  updateOnSelect: false,
```

Comment:
Do we need false if `undefined` is also falsey?

Reply:
Fair enough. I was very concerned about making it obvious this is opt-in. You are correct that it's unnecessary.